contracts/interfaces/README.md

```diff
@@ -0,0 +1,72 @@
+## Contracts interfaces
+
+Smart contract interfaces of the 0x protocol.
+
+## Usage
+
+Contracts that make up and interact with version 2.0.0 of the protocol can be found in the [contracts](./contracts) directory. The contents of this directory are broken down into the following subdirectories:
```

Comment:
Think we can remove the mention of `2.0.0` specifically in all these packages now.